package.json

```diff
@@ -8,6 +8,8 @@
   },
   "dependencies": {
     "@babel/polyfill": "7.4.4",
+    "@patternfly/react-core": "^3.75.2",
+    "@patternfly/react-styles": "^3.5.7",
```

Comment:
react-styles is brought as a dependency of react-core anyway. You should not explicitly set it unless some specific version is broken and you need to specify it.
See `npm info @patternfly/react-core` to verify that it's indeed dependency of that package.